docs/natspec-format.rst

```diff
@@ -236,6 +237,7 @@ file should also be produced and should look like this:
       "kind" : "dev",
       "author" : "Larry A. Gardner",
       "details" : "All function calls are currently implemented without side effects",
+      "custom:experimental": "This is an experimental contract.",
```

Comment:
@chriseth Actually, do custom tags only show up in devdocs?

Reply:
Currently, yes. Why should it be in userdoc?

Reply:
Because they are custom, could be used for anything, right?

Currently they are restricted to developer use cases. We should document it then.

Reply:
Just checked, it is already mentioned below the tag list.

Reply:
There is exactly one tag that is meant for users and that is `@notice`.